app/controllers/admin/users_controller.rb

```diff
@@ -3,6 +3,10 @@ class Admin::UsersController < Admin::BaseController
   def index
     authorize(User)
     @users = sort(policy_scope(User.public_send(user_role))).page(params[:page])
+
+    if params[:search]
+      @users = @users.email_search(params[:search][:search_term])
+    end
```

Comment:
That might cause you some performance issues if you have 10000 users in database and search for letter "a".

What I usually do is validating the minimum number of characters before performing the search, like:

```ruby
if params[:search].present? && if params[:search].size >= 3
  @users = @users.email_search(params[:search][:search_term])
end
```